### DISCUSSION TITLE
Error in connecting  Phidget 1048 and Mac OS 11

### DISCUSSION THREAD

**Discussion by cavixyeh** [artisan-roaster-scope/artisan#646] [Q&A]
Hi , 
I'm using Mac on OS 11 and Phidget had changed the way to install drivers  [Phidget for mac os 11 support](https://www.phidgets.com/docs/OS_-_macOS), so i followed the new one and use the standalone control panel of phidget to for my phidget 1048, it looks good in the phidget app ,having the temprature readings. but it seems artisan cannot get the correct connection in this way [Error log](https://photos.app.goo.gl/5xw7ZJq2wnmMXgXv5).

Am I missing something to setup the connection in Artisan ??

**Comment by MAKOMO** [comment]
The link you provided installs a standalone version of the Phidgets Control Panel that comes with the Phidget driver build in. However, that included driver can only be used by that one application it got integrated with, the Phidgets Control Panel. So basically there is no Phidget driver installed on your machine as indicated by the Error log you provided. Artisan depends on a system wide install of the Phidgets driver. Just install the regular Phidget driver and you are good to go.
